client/internal/routemanager/dynamic/route.go

```diff
@@ -228,11 +235,16 @@ func (r *Route) resolve(results chan resolveResult) {
 		wg.Add(1)
 		go func(domain domain.Domain) {
 			defer wg.Done()
-			ips, err := net.LookupIP(string(domain))
+
+			ips, err := r.getIPsFromResolver(domain)
 			if err != nil {
```

Comment:
We should log this error, and probably drop the `log.Debugf`s within `getIPsFromResolver` in the `err != nil` clauses